src/style.css

```diff
@@ -90,7 +90,7 @@ p {
 }
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages capitalized to make your pull request descriptive and professional 👍